### PR TITLE
Saying which table is effected by Error

### PR DESCRIPTION
**Pull request by mogwai**

### CODE REVIEW

File: lib/Table.js [hunk 1]
```diff
@@ -162,8 +162,9 @@ Table.prototype.init = function (next) {
 
         } else {
           if (indexes.delete.length > 0 || indexes.create.length > 0) {
-            debug('indexes are not synchronized and update flag is set to false');
-            deferred.reject(new Error('indexes are not synchronized and update flag is set to false'));
+            var mess = table.name + 'indexes are not synchronized and update flag is set to false';
```
Comment (fishcharlie) [inline lib/Table.js]: Can we add a space before the `indexes` word so there is a space between the table name and indexes?

**Comment by fishcharlie** [comment]
@mogwai For that last commit, isn't that pretty similar to #331?

**Comment by mogwai** [comment]
Yup shall I remove it?

**Comment by fishcharlie** [comment]
@mogwai Yeah that'd be great.

**Pull request merged**